### DISCUSSION TITLE
How to add image.gif in a file markdown and can resize the image.gif

### DISCUSSION THREAD

**Discussion by SavantEVO** [squidfunk/mkdocs-material#6326] [Q&A] [answered]
I tried to add gif in file.md and I need to resize the gif. How can file.md support option resize gif ? or Help me some solution to fix resize ?
```
## The Workflow :robot:
![Alt Text](../assets/images/Workflow_BecaStreaming.gif)
```

**Comment by alexvoss** [comment, marked as answer]
As the [documentation on using and formatting images](https://squidfunk.github.io/mkdocs-material/reference/images) suggests, the [Attribute List extension](https://squidfunk.github.io/mkdocs-material/setup/extensions/python-markdown/#attribute-lists) can be used to style images. You could specify the width, for example, or add padding. Best to do so with a CSS class and some [extra_css](https://squidfunk.github.io/mkdocs-material/customization#additional-css):

```
![Image title](https://dummyimage.com/600x400/eee/aaa){ .image25percent }
```
